nix/modules/holonix-integration-test.nix

```diff
@@ -58,6 +59,8 @@
           # output to console and to logfile
           exec >> >(tee $out) 2>&1
 
+          eval "$shellHook"
```

Comment:
Not understanding what this is for? It's after the tests?

Reply:
the tests start below where `${testScript`} is placed. locally i got a version that unpacks that variable into the string here as the indirection isn't helpful

Reply:
done; i find it easier to follow now. PTAL!